applications/resturant/serializers/core.py

```
@@ -48,6 +48,24 @@
         raise serializers.ValidationError("Party size must be between 1 and 25.")
 
 
+def price_is_valid(value: float) -> Union[None, serializers.ValidationError]:
+    """Check if the price is a positive number.
+    Args:
+        value (float): The price of the menu item
+    """
+    if value <= 0:
+        raise serializers.ValidationError("Price must be a positive number.")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)